app/models/plan.rb

```diff
@@ -28,20 +28,24 @@
 #  template_id                       :integer
 #  org_id                            :integer
 #  funder_id                         :integer
+#  grant_id                          :integer
+#  api_client_id                     :integer
 #
 # Indexes
 #
-#  index_plans_on_template_id  (template_id)
-#  index_plans_on_funder_id    (funder_id)
+#  index_plans_on_template_id   (template_id)
+#  index_plans_on_funder_id     (funder_id)
+#  index_plans_on_grant_id      (grant_id)
+#  index_plans_on_api_client_id (api_client_id)
 #
 # Foreign Keys
 #
 #  fk_rails_...  (template_id => templates.id)
 #  fk_rails_...  (org_id => orgs.id)
 #
 
-# TODO: Drop the funder_name column once the funder_id has been back
-#       filled and we're removing the is_other org stuff
+# TODO: Drop the funder_name and grant_number columns once the funder_id has
+#       been back filled and we're removing the is_other org stuff
```

Comment:
We can do a pass through the code for all these TODO's after the Rails 5 migration.
Thanks for keeping these up to date